src/common/models/modifier.ts

```diff
@@ -6,31 +6,31 @@ export interface Modifiers {
 export interface Modifier {
   label: string;
   total: number;
-  id: ModifierTypes;
+  id: string;
 }
 
 export enum ModifierTypes{
-  STR,
-  DEX,
-  CON,
-  INT,
-  WIS,
-  CHA
+  STR = 'STR',
+  DEX = 'DEX',
+  CON = 'CON',
+  INT = 'INT',
+  WIS = 'WIS',
+  CHA = 'CHA'
```

Comment:
Assigned an actual value to these. Otherwise it was causing issues. Still learning proper use of enum's